docs/fleet/cli.md

```diff
@@ -363,10 +369,20 @@ viam data database hostname --org-id=abc
 | `export`      | export data in a specified format to a specified location  | - |
 | `database configure`      | create a new database user for the Viam organization's MongoDB Atlas Data Federation instance, or change the password of an existing user. See [Configure data query](/data/query/#configure-data-query)  | - |
 | `database hostname`      | get the MongoDB Atlas Data Federation instance hostname and database name. See [Configure data query](/data/query/#configure-data-query)  | - |
+| `dataset add`      | add a new image to an existing dataset by its file id, or add a group of images by specifying a filter | `filter` |
+| `dataset remove`      | remove an existing image from a dataset by its file id, or remove a group of images by specifying a filter | `filter` |
 | `delete binary`      | delete binary data  | - |
 | `delete tabular`      | delete tabular data  | - |
 | `--help`      | return help      | - |
 
+##### Positional arguments: `database`
```

Comment:
* We have many positional arguments within the top-level `data` command (like 'data delete binary --etc' or 'data delete tabular --etc'), which might mean that we should file a separate ticket to document the other positional arguments available.
* The below `filter` is for `dataset` rather than `database`. 
* Related to adding `ids`, `filter` is one option for adding or removing, but you could also add or remove by `ids` as a positional argument. Both of the following are valid:

```
viam data dataset add filter --dataset-id=abc --location-id=123 --org-id=123 --start=2023-01-01T05:00:00.000Z --end=2023-10-01T04:00:00.000Z --tags=example
```
and

```
viam data dataset add ids --dataset-id=abc --location-id=123 --org-id=123 --file-ids=one,two,three
```

Reply:
Thank you, I updated the header, was a typo, and have added a new section on `ids`, based on what you included in your comment.

If there are more positional parameters, please do create a new ticket for anything not presently covered in this PR, and I will be happy to move on to that ticket next! Thank you!